packages/UI/src/components/organisms/cards/PositionCard/PositionCard.stories.tsx

```diff
@@ -13,10 +13,11 @@ const TemplatePositionCard: ComponentStory<typeof PositionCard> = (args) => (
 const content = {
   title: 'Web Developer',
   level: 'junior',
-  timeCommittment: '5 hrs per week',
+  commitmentHoursPerWeek: '5 hrs per week',
```

Comment:
why changing those? changing those can result in broken components that use the old props, please revert this change